README.md

```diff
@@ -136,3 +140,242 @@ After editing any schema, you have to create new database migrations to apply th
 The comment will go in the filename, so it should really be short.  
 Look out for any warnings, and review the created migration file before applying it (with `alembic upgrade head`).
 
+## Usage:
+
+Here we give some instructions for using and extending the code. 
+
+### Exposures and data files
+
+An Exposure object is a reference to the raw data taken by an instrument/telescope
+at one point in time and space. It can include multiple images if the instrument
+has multiple sections, e.g., for a mosaic camera, or a multi-channel instrument, 
+e.g., using a dichroic. 
+
+The Exposure must have some basic properties such as the time of observation
+(which is represented as the modified Julian date, or MJD when the image was taken), 
+the coordinates the telescope was pointing at (RA and Dec),
+and some housekeeping information like the project/propsal under which the images were taken, 
+the name of the object/target/field that was observed, 
+the exposure time, and the filter used (or filter array). 
+
+The Exposure object will contain the filename and path to the underyling raw data files. 
+Ideally these data files are not modified at all from when they were generated at the telescope. 
+Additional processing steps, such as bias/flat should be done on the next processing level
+(i.e., when generating Image objects) so the details of the processing can be tracked.
+The Exposure's `filename` column will contain information on the name and path of the data
+files relative to some data root folder. The root folder can be changed using the config
+(e.g., when moving a mount point) and can contain multiple root folders for storing
+data on a server and locally. The `filename` is always relative to that, so the database
+entry remains valid even if the data is moved.
+For multiple-section instruments, there would be multiple data arrays saved to disk. 
+These can be saved in one file (e.g., using a multi-extension FITS file) or in multiple files.
+In that case, the `filename_extensions` will contain an array of strings, 
+each containing the last part of the filename that completes the partial filename in `filename`.
+These can be normal file extensions:
+
+```
+filename = 'Camera_Project_2021-01-01T00:00:00.000.fits'
+filename_extensions = ['.bias', '.flat'] 
```

Comment:
I guess this is a bad example because I used bias/flat. Those would also have the same `filename_extensions` array, but I will give a better example.